Resources/Prototypes/_Goobstation/Entities/Mobs/Customization/Markings/yowie_fur.yml

```
@@ -0,0 +1,46 @@
+      type:
+        !type:SimpleColoring
+          color: "#FFFFFF"
+  sprites:
+  - sprite: _Goobstation/Mobs/Customization/yowie_fur.rsi
+    state: yowie_fur
+    
+- type: marking
+  id: YowieFurFeetRight
+  bodyPart: RFoot
+  markingCategory: Overlay
+  speciesRestriction: [Yowie] 
```

yamllint

[error] trailing spaces

(trailing-spaces)